src/commands/help.js

```diff
@@ -1,44 +1,48 @@
 const Discord = require('discord.js');
+const fs = require('fs');
 module.exports = {
   prefix: '!help',
+  description: 'Get the commands currently available with this bot',
   /**
    * @name help
    * Displays currently available commands.
    *
    * @param {Discord.Message} message the message provided
-   *Use this format to add new commands:
-   *{name: "Command Name", value: "Command Description"},
+   * Now automatically adds new commands - make sure the .js file has a prefix and description,
    */
 
   command: async function help(message) {
     //console.log("help", message);
     try {
-      const helpEmbed = new Discord.MessageEmbed()
-        .setColor('#0099ff')
-        .setTitle('Bot Information')
-        .setDescription(
-          'Hello! I am a test bot created by bradtaniguchi and members of FreeCodeCamp.org ' +
-            'to experiment with the process of building a Discord Bot. You can view my source ' +
-            'code at https://github.com/bradtaniguchi/discord-bot-test'
-        )
-        .addFields(
-          { name: '\u200B', value: '\u200B' },
-          {
-            name: 'Commands',
-            value: 'Here are my currently available commands!'
-          },
-          {
-            name: '!eightball <optional string>',
-            value: 'Returns a response from a magic pool ball!'
-          },
-          //new command lines go below here.
+      const helpEmbed = {
+        color: '#0099FF',
+        title: 'Bot Information',
+        description:
+          'Hello! I am a test bot created by bradtaniguchi and members of FreeCodeCamp.org to experiment with the process of building a Discord Bot. You can view my source code at https://github.com/bradtaniguchi/discord-bot-test',
+        fields: [],
+        footer: { text: 'I am not affiliated with FreeCodeCamp in any way.' }
+      };
 
-          //for spacing, leave this one at bottom.
-          { name: '\u200B', value: '\u200B' }
-        )
-        .setFooter('I am not affiliated with FreeCodeCamp in any way.');
+      fs.readdir(__dirname, (error, files) => {
+        if (error) {
+          console.error(error);
+        }
+        for (let i = 0; i < files.length; i++) {
+          let filename = files[i];
+          let lookup = require(`./${filename}`);
+          if (lookup.prefix == undefined || lookup.description == undefined) {
```

Comment:
Don't use `==`, use `===`, or just check for "falsy":
```js
if (!lookup.prefix || !lookup.description) {
```

Reply:
Done :)